README.md

```diff
@@ -12,5 +12,17 @@ This is a demo of the various [Meteoric](https://github.com/meteoric) packages:
 
 See it in action here: [meteoric.github.io](http://meteoric.github.io)
 
+## Running locally
+
+Due to [some](https://github.com/meteoric/demo/issues/4) [issues](https://github.com/meteoric/demo/issues/6), you need to run the following for a fully-functional demo:
+
+```sh
+git clone git@github.com:meteoric/demo.git
+meteor run  # will fail
+^C
+meteor add meteoric:ionic@0.1.17
```

Comment:
OK, this is just my first encounter with the project. As long as something's added so new users don't bounce, that should be good.